hnn_core/gui/viz_manager.py

```diff
@@ -0,0 +1,430 @@
+        value=3000,
+        description='Dipole Scaling:',
+        disabled=False, layout=layout,
+        style=analysis_style)
+
+    max_spectral_frequency = FloatText(
+        value=100,
+        description='Max Spectral Frequency (Hz):',
+        disabled=False, layout=layout,
+        style=analysis_style)
+
+    clear_button = Button(description='Clear axis')
+    clear_button.on_click(partial(_clear_axis, ax=ax))
+
+    plot_button = Button(description='Plot')
+    plot_button.on_click(
+        partial(
+            _plot_on_axes,
+            widgets_simulation=simulation_selection,
+            widgets_plot_type=plot_type_selection,
+            spectrogram_colormap_selection=spectrogram_colormap_selection,
+            dipole_smooth=dipole_smooth,
+            max_spectral_frequency=max_spectral_frequency,
+            dipole_scaling=dipole_scaling,
+            data=data,
+            fig=fig,
+            ax=ax,
+        ))
+
+    vbox = VBox([
+        simulation_selection, plot_type_selection, dipole_smooth,
+        dipole_scaling, max_spectral_frequency, spectrogram_colormap_selection,
+        HBox(
+            [plot_button, clear_button],
+            layout=Layout(justify_content='space-between', ),
+        )
+    ], layout=Layout(width="100%"))
+
+    return vbox
+
+
+def _close_figure(b, widgets, data, fig_idx):
+    fig_related_widgets = [widgets['figs_tabs'], widgets['axes_config_tabs']]
+    for w_idx, tab in enumerate(fig_related_widgets):
+        tab_children = list(tab.children)
+        titles = [tab.get_title(idx) for idx in range(len(tab.children))]
+        tab_idx = titles.index(_idx2fname(fig_idx))
+        print(f"Del fig_idx={fig_idx}, fig_idx={fig_idx}")
+        del tab_children[tab_idx], titles[tab_idx]
+
+        tab.children = tuple(tab_children)
+        [tab.set_title(idx, title) for idx, title in enumerate(titles)]
+
+        if w_idx == 0:
+            plt.close(data['figs'][fig_idx])
+            del data['figs'][fig_idx]
+            n_tabs = len(tab.children)
+            for idx in range(n_tabs):
+                _fig_idx = _fname2idx(tab.get_title(idx))
+                assert _fig_idx in data['figs'].keys()
+
+                tab.children[idx].clear_output()
+                with tab.children[idx]:
+                    display(data['figs'][_fig_idx].canvas)
+
+        if n_tabs == 0:
+            widgets['figs_output'].clear_output()
+            with widgets['figs_output']:
+                display(Label(_fig_placeholder))
+
+
+def _add_axes_controls(widgets, data, fig, axd):
+    fig_idx = data['fig_idx']['idx']
+
+    controls = Tab()
+    children = [
+        _get_ax_control(widgets, data, fig_idx=fig_idx, fig=fig, ax=ax)
+        for ax_key, ax in axd.items()
+    ]
+    controls.children = children
+    for i in range(len(children)):
+        controls.set_title(i, f'ax{i}')
+
+    close_fig_button = Button(description=f'Close {_idx2fname(fig_idx)}',
```

Comment:
something funky is happening with this button if you add many figures ... the text seems to shift right:

![image](https://user-images.githubusercontent.com/15852194/186487750-7d51405f-252d-4547-b5e2-e53cc873b996.png)

Reply:
It seems this is a known issue. See this: https://github.com/jupyter-widgets/ipywidgets/issues/1803.
We can add `overflow-x:scroll` to allow users to scroll but it will still be a bit ugly.